vector-config/src/main/res/values/config-settings.xml

```diff
@@ -28,6 +28,8 @@
 
     <bool name="settings_interface_bubble_visible">true</bool>
     <bool name="settings_interface_bubble_default">false</bool>
+    <bool name="settings_presence_visible">false</bool>
+    <bool name="settings_presence_offline_mode_default">false</bool>
```

Comment:
This feels worthless, I wonder if we could simply pass on this, since it kinda doubles on the default value in MatrixConfiguration 🤔